docs/tutorial/01-simple-template.md

```diff
@@ -8,9 +8,20 @@ If you haven't already, follow [these instructions](../installing.md) to install
 
 Let's start by creating a blank file `main.bicep` and compiling it by running:
 
-```bash
-bicep build main.bicep
-```
+
+1. Create the blank main.bicep file
+  - Windows
+    ```bash
+    fsutil file createnew main.bicep 0
```

Comment:
TIL about `fsutil` :) 

Is there a single canonical way to do this on windows? seems like there are a lot of options and recommendations out there

Reply:
No there isn’t. But most of the options are here: https://blog.jongallant.com/2016/11/windows-touch-equivalent/

I like fsutil because it creates a truly blank file, doesn’t require any additional install or bash, and doesn’t have a false negative error message when run